.flake8

```diff
@@ -0,0 +1,25 @@
+[flake8]
+select = E,F,W
+exclude = ./build
+ignore =
```

Comment:
Flake8一共有113个子项
- 错误码E、W：19（E1）+25（E2）+6（E3）+2（E4）+2（E5）+14（E7）+2（E9）+1（W1）+3（w2）+1（w3）+3（W5）-11（*）=67
- 错误码F：7（F4）+15（F5）+8（F6）+9（F7）+6（F8）+1（F9）=46

目前Paddle代码中存在的问题（ignore部分）是69个。

请确认下上述数字。

Reply:
> W6（6 个）貌似是漏掉了

确实是漏掉了。。

> (*) In the default configuration, the checks E121, E123, E126, E133, E226, E241, E242, E704, W503, W504 and W505 are ignored because they are not rules unanimously accepted,

可以先不修复默认配置里不包含的11个，因为这11个并没有得到广泛认可。

> 如果加上 6 个 W6、E999、C901 一共 121 个

对的，以这个数为准。

> 加上 E999 之前存在 70 个，已修复 E999（语法错误）、W291、W293（空格错误）三个错误码，目前存在 67 个错误码，在当前配置中 ignore 掉了

一共70个子项，已完成3个。

Reply:
> 之后将会优先修复一些对代码风格影响较大的错误码（比如 F401）

有错误码对应的数量统计么？